src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -95,7 +97,28 @@ public class PPLTool implements Tool {
 
     private static Map<String, String> defaultPromptDict;
 
+    private static Set<String> allowedFieldsType;
```

Comment:
Static variables usually follows capital underscore separated naming convention, e.g. `ALLOWED_FIELDS_TYPE`, please change this one as well as the defaultPromptDict one. Also make them to final since they shouldn't be changed anywhere.

Reply:
Sure